src/sysext/sysext.c

```diff
@@ -169,6 +236,9 @@ static int unmerge_hierarchy(const char *p) {
 
 static int unmerge(void) {
         int r, ret = 0;
+        bool reload = false;
```

Comment:
here you are using an auxiliary variable and below you are not - keep it consistent please, and just do the check inline